crates/dreammaker/src/constants.rs

```diff
@@ -812,6 +815,12 @@ impl<'a> ConstantFolder<'a> {
                         None => return Err(self.error("malformed nameof() call, expression appears to have no name")),
                     }
                 }
+                "fexists" => {
+                    if args.len() != 1 {
+                        return Err(self.error(format!("malformed fexists() call, must have 1 argument and instead has {}", args.len())));
+                    }
+                    Constant::Call(ConstFn::FileExists, self.arguments(args)?)
```

Comment:
I suggest:
1. Like `"defined"`, requiring `self.defines.is_some()`, so that this function will be properly rejected if used in a type var context.
2. Like `"defined"`, actually check whether the file exists and evaluate to 0 or 1 here. Otherwise a `Constant::Call(..)` is always considered truthy (see `Constant::to_bool`) and `#if fexists("anything")` will always pass.

Reply:
should be addressed